ci/test.yml

```diff
@@ -269,20 +268,46 @@ connect-popup legacy:
     <<: *run_everything_rules
   when: manual
 
-connect-popup-webextension:
-  extends: .e2e connect-popup
+.e2e connect-popup-webextension:
+  stage: integration testing
+  retry: 1
   variables:
+    COMPOSE_PROJECT_NAME: $CI_JOB_ID
+    COMPOSE_FILE: ./docker/docker-compose.connect-popup-ci.yml
+    URL: ${DEV_SERVER_URL}/connect/${CI_COMMIT_REF_NAME}/
+    TEST_FILE: $TEST_FILE
     IS_WEBEXTENSION: "true"
+  script:
+    - yarn install --immutable
+    - yarn workspace @trezor/connect-explorer-webextension build
+    - docker-compose pull
+    - docker-compose up -d trezor-user-env-unix
+    - docker-compose run test-run
+  after_script:
+    - docker-compose down
+    - docker network prune -f
+  artifacts:
+    expire_in: 7 days
+    when: always
+    paths:
+      - ./packages/connect-popup/e2e/screenshots
+      - ./packages/connect-popup/connect-popup-overview.html
+      - ./packages/connect-popup/test-results
+  interruptible: true
+  parallel:
+    matrix:
+      - TEST_FILE: ["methods.test"]
```

Comment:
this is a good start but we are interested also in other testing suites. for example "popup-close" where there is different implementation of popup closing and opening from connect on web. 

it can be a followup, but lets not forget about it.